### PR TITLE
MAGECLOUD-1806: Add Docker configuration to template

### PR DESCRIPTION
**Pull request by shiftedreality**
### Related PR
- https://github.com/magento/ece-tools/pull/198

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+app/etc/config.php
+app/etc/env.php
+vendor
+generated
+pub/static
+pub/media
+var/view_preprocessed
+.git
```
Comment (bbatsche) [inline .dockerignore]: I believe `.dockerignore` is only useful when building images, which should be done from the `cloud-docker` repository, so this file shouldn't belong here

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+services:
+  fpm:
+    hostname: fpm.magento2.docker
+    image: shiftedreality/cloud-docker-php:7.1-fpm
+    ports:
+      - 9000
+    links:
+      - db
+    volumes_from:
+      - appdata
```
Comment (bbatsche) [inline docker-compose.yml]: Docker Compose's `volume` syntax should eliminate the need for an additional third party containers or using `volumes_from` here

Comment (bbatsche) [reply docker-compose.yml]: https://docs.docker.com/compose/compose-file/compose-file-v2/#volumes

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      - fpm
+      - db
+    volumes_from:
+      - appdata
+    env_file:
+      - ./docker/global.env
+#    environment:
+#      - VIRTUAL_HOST=magento2.docker
+#      - VIRTUAL_PORT=8080
+#      - HTTPS_METHOD=noredirect
```
Comment (bbatsche) [inline docker-compose.yml]: Did you mean to include commented lines?

Comment (shiftedreality) [reply docker-compose.yml]: Yes, this is custom config which require changes to `.hosts` file

File: .dockerignore (hunk 1 above)
Comment (shiftedreality) [reply .dockerignore]: Agree, removed

File: docker-compose.yml (hunk 2 above)
Comment (shiftedreality) [reply docker-compose.yml]: It's using to avoid copy-paste

Comment (shiftedreality) [reply docker-compose.yml]: @JacobBrownAustin you cannot have service without an image. They're using in few places, so that's why separated.

**Comment by YPyltiai** [comment]
@shiftedreality , can we close this one?

**Pull request closed without merging**